services/static-webserver/client/source/class/osparc/utils/DisabledPlugins.js

```diff
@@ -57,6 +57,10 @@ qx.Class.define("osparc.utils.DisabledPlugins", {
       return this.__isPluginDisabled(this.LICENSES);
     },
 
+    isSimultaneousAccessEnabled: function() {
+      return false;
```

Comment:
See examples  I left in the `How to test` section of https://github.com/ITISFoundation/osparc-simcore/pull/8120 to implement this